index.rst

```diff
@@ -265,10 +265,17 @@ Welcome to PyTorch Tutorials
    :tags: Production,ONNX,Backends
 
 .. customcarditem::
-   :header: Introduction to ONNX Registry
-   :card_description: Demonstrate end-to-end how to address unsupported operators by using ONNX Registry.
+   :header: Extending the ONNX exporter operator support
+   :card_description: Demonstrate end-to-end how to address unsupported operators in ONNX.
    :image: _static/img/thumbnails/cropped/Exporting-PyTorch-Models-to-ONNX-Graphs.png
-   :link: advanced/onnx_registry_tutorial.html
+   :link: beginner/onnx_registry_tutorial.html
```

Comment:
this link is 404: https://pytorch.org/tutorials/beginner/onnx_registry_tutorial.html
```suggestion
   :link: beginner/onnx/onnx_registry_tutorial.html
```